test/__fixtures/BaseTest.t.sol

```diff
@@ -16,11 +16,13 @@ import { ReservoirPriceOracle, PriceType, IPriceOracle } from "src/ReservoirPric
 contract BaseTest is Test {
     using FactoryStoreLib for GenericFactory;
 
+    uint64 internal constant DEFAULT_REWARD_GAS_AMOUNT = 200_000;
+
     GenericFactory internal _factory = new GenericFactory();
     ReservoirPair internal _pair;
 
     ReservoirPriceOracle internal _oracle =
-        new ReservoirPriceOracle(0.02e18, 15 minutes, 500_000, PriceType.CLAMPED_PRICE);
+        new ReservoirPriceOracle(0.15 minutes, DEFAULT_REWARD_GAS_AMOUNT, PriceType.CLAMPED_PRICE);
```

Comment:
nope haha, dis a product of insufficient backspacing . I've refactored it to use a named constant to prevent something like that from happening again.